### PR TITLE
formulae: add more licenses

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

#58225

### CODE REVIEW

File: Formula/aacgain.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Aacgain < Formula
   # See: https://github.com/Homebrew/homebrew/issues/16838
   url "https://aacgain.altosdesign.com/alvarez/aacgain-1.8.tar.bz2"
   sha256 "2bb8e27aa8f8434a4861fdbc70adb9cb4b47e1dfe472910d62d6042cb80a2ee1"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/aacgain.rb:10]: Shouldn't this be `GPL-2.0-only` as per https://github.com/Homebrew/homebrew-core/issues/58225#issuecomment-667874798?

File: Formula/abcl.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ class Abcl < Formula
   homepage "https://abcl.org/"
   url "https://abcl.org/releases/1.7.1/abcl-src-1.7.1.tar.gz"
   sha256 "d51014b2be6ecb5bcaaacda0adf4607a995dd4b6e9e509c8a1f5a998b7649227"
+  license "GPL-2.0-with-classpath-exception"
```
Comment (miccal) [inline Formula/abcl.rb:6]: This one is on the list of [Deprecated License Identifiers](https://spdx.org/licenses/GPL-2.0-with-classpath-exception.html).

File: Formula/avrdude.rb [hunk 3]
```diff
@@ -4,6 +4,7 @@ class Avrdude < Formula
   url "https://download.savannah.gnu.org/releases/avrdude/avrdude-6.3.tar.gz"
   mirror "https://download-mirror.savannah.gnu.org/releases/avrdude/avrdude-6.3.tar.gz"
   sha256 "0f9f731b6394ca7795b88359689a7fa1fba818c6e1d962513eb28da670e0a196"
+  license "GPL-2.0+"
```
Comment (miccal) [inline Formula/avrdude.rb:7]: `GPL-2.0-or-later`

File: Formula/bfg.rb [hunk 4]
```diff
@@ -3,6 +3,7 @@ class Bfg < Formula
   homepage "https://rtyley.github.io/bfg-repo-cleaner/"
   url "https://search.maven.org/remotecontent?filepath=com/madgag/bfg/1.13.0/bfg-1.13.0.jar"
   sha256 "bf22bab9dd42d4682b490d6bc366afdad6c3da99f97521032d3be8ba7526c8ce"
+  license "GPL-3.0+"
```
Comment (miccal) [inline Formula/bfg.rb:6]: `GPL-3.0-or-later`

File: Formula/brag.rb [hunk 5]
```diff
@@ -3,6 +3,7 @@ class Brag < Formula
   homepage "https://brag.sourceforge.io/"
   url "https://downloads.sourceforge.net/project/brag/brag/1.4.3/brag-1.4.3.tar.gz"
   sha256 "f2c8110c38805c31ad181f4737c26e766dc8ecfa2bce158197b985be892cece6"
+  license "GPL-2.0+"
```
Comment (miccal) [inline Formula/brag.rb:6]: `GPL-2.0-or-later`

File: Formula/brogue.rb [hunk 6]
```diff
@@ -6,6 +6,7 @@ class Brogue < Formula
   url "https://sites.google.com/site/broguegame/brogue-1.7.5-linux-amd64.tbz2"
   version "1.7.5"
   sha256 "a74ff18139564c597d047cfb167f74ab1963dd8608b6fb2e034e7635d6170444"
+  license "AGPL-3.0"
```
Comment (miccal) [inline Formula/brogue.rb:9]: `AGPL-3.0-only`

File: Formula/byteman.rb [hunk 7]
```diff
@@ -3,6 +3,7 @@ class Byteman < Formula
   homepage "https://byteman.jboss.org/"
   url "https://downloads.jboss.org/byteman/4.0.12/byteman-download-4.0.12-bin.zip"
   sha256 "7aebafd6877058a1406e725be4246bcafd8efd78fa583b7192e847cb5d6b27a5"
+  license "LGPL-2.1+"
```
Comment (miccal) [inline Formula/byteman.rb:6]: `LGPL-2.1-or-later`

File: Formula/bzr-builder.rb [hunk 8]
```diff
@@ -3,6 +3,7 @@ class BzrBuilder < Formula
   homepage "https://launchpad.net/bzr-builder"
   url "https://launchpad.net/bzr-builder/trunk/0.7.3/+download/bzr-builder-0.7.3.tar.gz"
   sha256 "9f8a078eafd6700ccbefa4e7e3f7df3240e15a2003c9538135c4be945ac90c91"
+  license "GPL-3.0"
```
Comment (miccal) [inline Formula/bzr-builder.rb:6]: `GPL-3.0-only`

File: Formula/bzr-externals.rb [hunk 9]
```diff
@@ -3,6 +3,7 @@ class BzrExternals < Formula
   homepage "https://launchpad.net/bzr-externals"
   url "https://launchpad.net/bzr-externals/trunk/1.3.3/+download/bzr-externals-1.3.3.tar.gz"
   sha256 "4ee33852f415cd5a982c4fa0bb548f86f2ae52368407c3e8d03d55bcfd7f3332"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/bzr-externals.rb:6]: `GPL-2.0-only`

File: Formula/bzr-extmerge.rb [hunk 10]
```diff
@@ -3,6 +3,7 @@ class BzrExtmerge < Formula
   homepage "https://launchpad.net/bzr-extmerge"
   url "https://launchpad.net/bzr-extmerge/trunk/1.0.0/+download/bzr-extmerge-1.0.0.tar.gz"
   sha256 "1b86d3a54fe669db19bc2a42a09eae52e449cc3ece8234377fd213e834f69cc0"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/bzr-extmerge.rb:6]: `GPL-2.0-only`

File: Formula/bzr-rewrite.rb [hunk 11]
```diff
@@ -3,6 +3,7 @@ class BzrRewrite < Formula
   homepage "https://launchpad.net/bzr-rewrite"
   url "https://launchpad.net/bzr-rewrite/trunk/0.6.3/+download/bzr-rewrite-0.6.3.tar.gz"
   sha256 "f4d0032a41a549a0bc3ac4248cd4599da859174ea33e56befcb095dd2c930794"
+  license "GPL-3.0"
```
Comment (miccal) [inline Formula/bzr-rewrite.rb:6]: `GPL-3.0-only`

File: Formula/bzr-upload.rb [hunk 12]
```diff
@@ -3,6 +3,7 @@ class BzrUpload < Formula
   homepage "https://launchpad.net/bzr-upload"
   url "https://launchpad.net/bzr-upload/1.1/1.1.0/+download/bzr-upload-1.1.0.tar.gz"
   sha256 "a48fc56d83114d9ab946cc358a5b33cb05e134787be135eb0a499317d6dec7fc"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/bzr-upload.rb:6]: `GPL-2.0-only`

File: Formula/bzr-xmloutput.rb [hunk 13]
```diff
@@ -3,6 +3,7 @@ class BzrXmloutput < Formula
   homepage "https://launchpad.net/bzr-xmloutput"
   url "https://launchpad.net/bzr-xmloutput/trunk/0.8.8/+download/bzr-xmloutput-0.8.8.tar.gz"
   sha256 "73b9b2f6ce4d9910031df7fd153d56d14f833c20a106f099bee5a33463f73b36"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/bzr-xmloutput.rb:6]: `GPL-2.0-only`

File: Formula/aalib.rb [hunk 14]
```diff
@@ -3,6 +3,7 @@ class Aalib < Formula
   homepage "https://aa-project.sourceforge.io/aalib/"
   url "https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz"
   sha256 "fbddda9230cf6ee2a4f5706b4b11e2190ae45f5eda1f0409dc4f99b35e0a70ee"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/aalib.rb:6]: `GPL-2.0-only`

File: Formula/abcmidi.rb [hunk 15]
```diff
@@ -3,6 +3,7 @@ class Abcmidi < Formula
   homepage "https://ifdo.ca/~seymour/runabc/top.html"
   url "https://ifdo.ca/~seymour/runabc/abcMIDI-2020.07.28.zip"
   sha256 "27ab5cb5f4cdd267208c0c41ee49fcfb8db21927c3553ea6d85a28645821de66"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/abcmidi.rb:6]: `GPL-2.0-only`

File: Formula/aoeui.rb [hunk 16]
```diff
@@ -3,6 +3,7 @@ class Aoeui < Formula
   homepage "https://code.google.com/archive/p/aoeui/"
   url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/aoeui/aoeui-1.6.tgz"
   sha256 "0655c3ca945b75b1204c5f25722ac0a07e89dd44bbf33aca068e918e9ef2a825"
+  license "GPL-2.0"
```
Comment (miccal) [inline Formula/aoeui.rb:6]: `GPL-2.0-only`

**Comment by jonchang** [comment]
Probably needs https://github.com/Homebrew/brew/pull/8211

**Comment by SeekingMeaning** [comment]
Thank you @miccal for the in-depth review ❤️ Your comments were helpful in updating this PR to match #59204

**Pull request closed without merging**